node/network/protocol/src/lib.rs

```diff
@@ -612,67 +613,46 @@ pub mod vstaging {
 	/// Network messages used by the statement distribution subsystem.
 	#[derive(Debug, Clone, Encode, Decode, PartialEq, Eq)]
 	pub enum StatementDistributionMessage {
-		/// A signed full statement under a given relay-parent.
+		/// A notification of a signed statement in compact form.
 		#[codec(index = 0)]
-		Statement(Hash, UncheckedSignedFullStatement),
-		/// Seconded statement with large payload (e.g. containing a runtime upgrade).
-		///
-		/// We only gossip the hash in that case, actual payloads can be fetched from sending node
-		/// via request/response.
-		#[codec(index = 1)]
-		LargeStatement(StatementMetadata),
-	}
-
-	/// Data that makes a statement unique.
-	#[derive(Debug, Clone, Encode, Decode, PartialEq, Eq, Hash)]
-	pub struct StatementMetadata {
-		/// Relay parent this statement is relevant under.
-		pub relay_parent: Hash,
-		/// Hash of the candidate that got validated.
-		pub candidate_hash: CandidateHash,
-		/// Validator that attested the validity.
-		pub signed_by: ValidatorIndex,
-		/// Signature of seconding validator.
-		pub signature: ValidatorSignature,
-	}
+		Statement(Hash, UncheckedSignedStatement),
 
-	impl StatementDistributionMessage {
-		/// Get fingerprint describing the contained statement uniquely.
-		pub fn get_fingerprint(&self) -> (CompactStatement, ValidatorIndex) {
-			match self {
-				Self::Statement(_, statement) => (
-					statement.unchecked_payload().to_compact(),
-					statement.unchecked_validator_index(),
-				),
-				Self::LargeStatement(meta) =>
-					(CompactStatement::Seconded(meta.candidate_hash), meta.signed_by),
-			}
-		}
-
-		/// Get the signature from the statement.
-		pub fn get_signature(&self) -> ValidatorSignature {
-			match self {
-				Self::Statement(_, statement) => statement.unchecked_signature().clone(),
-				Self::LargeStatement(metadata) => metadata.signature.clone(),
-			}
-		}
-
-		/// Get contained relay parent.
-		pub fn get_relay_parent(&self) -> Hash {
-			match self {
-				Self::Statement(r, _) => *r,
-				Self::LargeStatement(meta) => meta.relay_parent,
-			}
-		}
-
-		/// Whether this message contains a large statement.
-		pub fn is_large_statement(&self) -> bool {
-			if let Self::LargeStatement(_) = self {
-				true
-			} else {
-				false
-			}
-		}
+		/// A notification of a backed candidate being known by the
+		/// sending node, for the purpose of being requested by the receiving node
+		/// if needed.
+		#[codec(index = 1)]
+		BackedCandidateInv {
```

Comment:
Stupid question, that came up with the ticket already: what does the `Inv` stand for?

Reply:
It stands for 'Inventory'